README.md

```diff
@@ -349,6 +349,201 @@ foreign keys using `django.db.models.ForeignKey`. The different entities within
 the classes are linked by a means one-to-many, many-to-one, and many-to-many
 relationship.
 
+### Set Up a Python Virtual Environment with Django
```

Comment:
Since our README already has a section on setting up pipenv, these virtualenv instructions might be redundant

Reply:
Since you already have pipenv you can just use the command `pipenv lock -r > requirements.txt` to generate the requirements.txt.  Elastic beanstalk does not natively support pipenv so you need the requirements file.

Reply:
While our README does have instructions for setting up pipenv, the pipenv instructions are for installing rather than creating and putting your terminal in a virtual environment. That is why we figured to include this with the other instructions for deployment. I also think that it would be helpful to repeat (if it truly is), that way a user does not have to scroll up and down to find the virtual environment instructions and back to the deployment ones.

Reply:
This should use pipenv to create the virtualenv.